10_deploy_rook.sh

```diff
@@ -13,13 +13,15 @@ oc create -f common.yaml
 sed '/FLEXVOLUME_DIR_PATH/!b;n;c\          value: "\/etc/kubernetes\/kubelet-plugins\/volume\/exec"' operator-openshift.yaml > operator-openshift-modified.yaml
 sed -i 's/# - name: FLEXVOLUME_DIR_PATH/- name: FLEXVOLUME_DIR_PATH/' operator-openshift-modified.yaml
 oc create -f operator-openshift-modified.yaml
+sleep 120
 oc wait --for condition=ready  pod -l app=rook-ceph-operator -n rook-ceph --timeout=120s
 oc wait --for condition=ready  pod -l app=rook-ceph-agent -n rook-ceph --timeout=120s
 oc wait --for condition=ready  pod -l app=rook-discover -n rook-ceph --timeout=120s
 sed "s/useAllDevices: .*/useAllDevices: true/" cluster.yaml > cluster-modified.yaml
 sed -i 's/# port: 8443/port: 8444/' cluster-modified.yaml
 oc create -f cluster-modified.yaml
 oc create -f toolbox.yaml
+sleep 120
```

Comment:
Can we do a wait here instead of an arbitrary sleep?

I'm worried about how we'll handle this in the final solution e.g via kni-install, shouldn't we have inter-dependent operators launched via manifests that will just wait until the needed things are ready?

I guess we could to imperative things in the cluster bootstrap code, but I was assuming we'd be able to throw the manifests at the cluster and just wait on the desired state, like most other components work?